### PR TITLE
Clean <head> tags

### PR DESCRIPTION
**Pull request by Senko-Dev**
**What this does:**

* Add Full All Copy OG Tags
* Replace links
* Fix contact JS
* Clean Head mess
* Organize into sections
* Redo Credits page (sync page)

**Why?**

This PR was made in order to improve the readability of the code and improve developer performance.

**Extra enhancements down the line:**

* Add Non-AllCopy OG Tags
* Improve SEO
* Keep users on the site
* Make contacts page work

### CODE REVIEW

File: about/credits/index.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
...
 <html lang="en-US">
 
-<head><meta name="twitter:site" content="@AytchSoftware"><meta property="og:image" content="https://h-projects.github.io/assets/images/brand/loqo.png"><meta property="og:site_name" content="Aytch Software"><meta property="og:type" content="website" />
-	<!-- import over logic -->
-	<script src="../assets/index.js"></script>
+<head>
+	<!-- js -->
+	<script src="/assets/index.js"></script>
+	<!-- ga -->
 	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-169183883-1"></script>
```
Comment (Indexerrowaty) [inline about/credits/index.html:8]: Why do we need Google analytics.
It's useless for anyone and spies on people for no reason.

**Review by Indexerrowaty — COMMENTED** [review]
Imo everything lgtm but we should consider getting rid of Google Analitycs.
It's not going to give us any useful information and It will spy on people and make website load slower.
Slower loading times = worse SEO score.

File: about/credits/index.html (hunk 1 above)
Comment (Senko-Dev) [reply about/credits/index.html:8]: Google Analytics does actually help with SEO from what I understand, in that it sells all that data to google /s.

Comment (Indexerrowaty) [reply about/credits/index.html:8]: It doesn't if you don't use these analitycs for improving SEO.

Comment (Senko-Dev) [reply about/credits/index.html:8]: SEO is fucked because ASTK/CNAME.

Comment (Indexerrowaty) [reply about/credits/index.html:8]: SEO has nothing to do with CNAME.

Comment (Senko-Dev) [reply about/credits/index.html:8]: Site down = Shit SEO.

**Pull request merged**